### PR TITLE
Remove allocs monitoring, fix docs, fix parameter tests

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR:
 - Removes the allocation monitoring (a step towards #1766)
 - Fixes some doc strings for file includes
 - Fixes the parameter tests and includes them in the test suite.

### CODE REVIEW

**Comment by charleskawczynski** [comment]
I did these all together because the doc string kind of spans the two issues

File: test/parameter_tests.jl [hunk 1]
```diff
@@ -1,8 +1,6 @@
 using Test
 import CLIMAParameters as CP
-
-include("../src/utils/cli_options.jl")
-include("../parameters/create_parameters.jl")
+import ClimaAtmos as CA
 
 FT = Float64
```
Comment (nefrathenrici) [inline test/parameter_tests.jl:6]: I can't seem to comment on collapsed lines in the PR but the two lines below need the `CA.` qualifier
```suggestion
s = CA.argparse_settings()
default_args = CA.cli_defaults(s)
```
Could be combined into one line as well: `default_args = CA.cli_defaults(CA.argparse_settings())`

Comment (nefrathenrici) [reply test/parameter_tests.jl:6]: Otherwise, this looks good to me! Thanks @charleskawczynski for fixing the parameter tests.

**Review by nefrathenrici — CHANGES_REQUESTED** [review]

**Review by nefrathenrici — APPROVED** [review]

**Pull request merged**